### PR TITLE
CAMEL-13684: Upgraded corda to 4.1 and added missing ops methods in producer

### PR DESCRIPTION
**Pull request by pkrishnath**

### CODE REVIEW

File: components/camel-corda/pom.xml [hunk 1]
```diff
@@ -36,6 +36,20 @@
     <properties>
     </properties>
 
+    <repositories>
```
Comment (oscerd) [inline components/camel-corda/pom.xml]: Please don't use third party repository. It's a nightmare. We already passed through this with restlet. If it's not available in central we'll wait for it. But please avoid third party repository.

**Comment by oscerd** [comment]
@davsclaus can you give your advice on third party repo in this case?

File: components/camel-corda/pom.xml (hunk 1 above)
Comment (pkrishnath) [reply components/camel-corda/pom.xml]: I have already raised with R3 to release the corda-rpc:4.1 to maven central ... Hope this will be fixed soon

Comment (pkrishnath) [reply components/camel-corda/pom.xml]: @oscerd , I have not tested karaf feature .. Would you be able to guid me how to perform karaf feature?

Comment (pkrishnath) [reply components/camel-corda/pom.xml]: Thanks @oscerd .. Thats done .. All good

**Comment by davsclaus** [comment]
Yeah its better to wait for it to be on maven central

**Comment by pkrishnath** [comment]
> corda-rpc is in central now, but corda-serialization is still missing

Yes, I am in touch with them (R3 team) to publish remaining  jars

**Comment by pkrishnath** [comment]
@oscerd @davsclaus all sorted .. They have published all 4.1 jars to maven central and i have removed R3 repo details from pom.xml.

**Pull request closed without merging**